lightkurve/targetpixelfile.py

```diff
@@ -1740,7 +1762,7 @@ def mission(self):
         """'Kepler' or 'K2'. ('MISSION' header keyword)"""
         return self.get_keyword('MISSION')
 
-    def extract_aperture_photometry(self, aperture_mask='default', centroid_method='moments'):
+    def extract_aperture_photometry(self, aperture_mask='default', flux_method='default', centroid_method='moments'):
```

Comment:
I recommend making `flux_method="sum"` the default on this line.  This makes the default behavior explicit, and removes duplicate behavior (i.e. right now `flux_method="sum"` and `flux_method="default"` would do the same thing, better to just offer one way).

Reply:
Note: in the docstring below, we are going to want to add an entry for "flux_method" in the "Parameters" section. The entry should mention that three strings can be passsed, i.e. 'sum', 'median', and 'mean'.

Reply:
First part is complete.
Where are the docstrings for me to edit.

Reply:
The docstring is the block of comments that immediately follow this line, i.e. immediately below "def extract_aperture_photometry(...)".  There is a section in that block of comments called "Parameters" which we need to update if we add parameters.

Reply:
updated.